src/fundus/publishers/shared/euronews.py

```diff
@@ -28,8 +28,7 @@ def body(self) -> ArticleBody:
 
         @attribute
         def authors(self) -> List[str]:
-            key_path = ["NewsArticle", "author", "name"]
-            author_string = self.precomputed.ld.get_value_by_key_path(key_path)
+            author_string = self.precomputed.ld.xpath_search("NewsArticle/author/name", scalar=True)
```

Comment:
Is the `scalar=True` on purpose here? Shouldn't multiple author elements be possible as well?

Reply:
I just mimicked the previous behavior bit your right, this should also be possible to take multiple authors into account.